src/ui-kit/Button/Button.tsx

```diff
@@ -50,6 +50,9 @@ export const RegularButton = JSX<ButtonCommonProps>(
     onClick,
     ...rest
   }) => {
+    const currentTheme = globalThis.document.querySelector('[data-theme]');
```

Comment:
```suggestion
    const currentTheme = globalThis?.document?.querySelector('[data-theme]') || 'pc';
```
для ssr нужна проверка